Utilities/NPC.gd

```diff
@@ -101,15 +102,16 @@ func animate(_dir):
 			"Right":
 				$AnimationPlayer.play("Right2")
 
-func set_idle_frame(_dir):
-	match _dir:
-		"Down":
+func set_idle_frame(idle):
```

Comment:
Using `idle` as a variable name sounds odd. Could you keep the `_dir` name, or use something else more clear?